package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "repository": "git@github.com:betagouv/annuaire-api.git",
   "dependencies": {
-    "ajv": "^6.1.1",
+    "ajv": "^6.5.4",
```

Comment:
The `yarn.lock` should have been modified too, no?

Reply:
Si tu as raison ! Merci beaucoup. J'ai fait en sorte sur la production repose sur `yarn` pour la gestion des dépendances.